includes/frontend/class-wcmp-checkout.php

```diff
@@ -434,6 +441,36 @@ public static function splitShippingMethodString(string $shippingMethod): array
 
         return $split;
     }
+
+    /**
+     * Show delivery options also for shipments on backorder
+     * @return bool
+     */
+    private function shouldShowDeliveryOptions(): bool
+    {
+        // $backorderDeliveryOptions causes the options to be displayed also when product is in backorder
```

Comment:
Als je deze comment nodig vindt moet de constante hernoemd worden, dan is hij gewoon niet duidelijk genoeg. Doe dan `SETTINGS_SHOW_DELIVERY_OPTIONS_FOR_BACKORDERS`

Reply:
done